app/src/hooks/useNetworkFees.ts

```diff
@@ -8,26 +8,30 @@ import React from 'react'
 // Hooks
 import { useWalletState } from '../state/wallet'
 
+// Utils
+import Amount from '../utils/amount'
+
 // Types
 import { NetworkInfo } from '../constants/types'
 
 export const useNetworkFees = () => {
   // Wallet State
   const {
-    state: { tokenSpotPrices, networkFeeEstimates }
+    state: { tokenSpotPrices, networkFeeEstimates, defaultBaseCurrency }
   } = useWalletState()
 
   const getNetworkFeeFiatEstimate = React.useCallback(
     (network: NetworkInfo) => {
       if (!networkFeeEstimates[network.chainId]) {
         return ''
       }
-      return (
+      const calculatedValue = (
         Number(tokenSpotPrices[network.symbol]) *
         Number(networkFeeEstimates[network.chainId].gasFee)
       ).toString()
+      return new Amount(calculatedValue).formatAsFiat(defaultBaseCurrency)
```

Comment:
Writing from my phone so please make sure it's syntactically ok.

```suggestion
      return new Amount(tokenSpotPrices[network.symbol]).times(networkFeeEstimates[network.chainId].gasFee).formatAsFiat(defaultBaseCurrency)
```

Reply:
Good call! Updated this!